data/entities.json5

```diff
@@ -8508,8 +8524,9 @@
   },
   {
     "name": "Throtle",
+    "homepage": "https://throtle.io/",
     "categories": ["analytics"],
-    "domains": ["v12group.com"]
+    "domains": ["v12group.com", "thrtle.com"]
```

Comment:
this is intentional misspelling right? :)

Reply:
Correct, that's the domain that use. I've also been cross checking against other projects like [Better. ](https://better.fyi/trackers/thrtle.com/)